homeassistant/components/netatmo/sensor.py

```diff
@@ -144,26 +145,25 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
         def find_devices(data):
             """Find all devices."""
-            all_module_names = data.get_module_names()
+            all_module_infos = data.get_module_infos()
+            all_module_names = [e["module_name"] for e in all_module_infos.values()]
             module_names = config.get(CONF_MODULES, all_module_names)
             _dev = []
```

Comment:
I suggest we rename this to `entities`, since that's what it will hold.

Reply:
Sure, but it's really just a list of names used during the discovery process.

Reply:
I'm referring to the `_dev` variable. It will hold a list of entity instances.

Reply:
Ah, sure, I was confused by the highlighting. Much more explicit.